test/unit/generic_states_test.rb

```diff
@@ -286,59 +286,6 @@ def test_condition_onset_during_encounter
     @patient.record_synthea.verify
   end
 
-  def test_allergy_onset
```

Comment:
Just noticed that these got left out when I did the rebase. I'll make sure I add them back in with the other fixes.